src/main/java/net/minecraftforge/fmp/item/IItemSaw.java

```diff
@@ -0,0 +1,10 @@
+package net.minecraftforge.fmp.item;
+
+import net.minecraft.item.ItemStack;
+
+public interface IItemSaw
+{
```

Comment:
It's not a leftover, no. It should be moved to a capability, but it's basically an interface that represents anything that can cut a block into microblocks so that anybody can add their own saw and they are all compatible (the microblocks themselves aren't included in the PR for obvious reasons, but in the case that there are two or more microblock mods, this would also be good for inter-compatibility).